docsteady/templates/dm-spec.latex.jinja2

```diff
@@ -216,6 +216,111 @@ Version & Status & Priority & Verification Type & Owner
 {% endfor %}
 {% endif %}
 
+{% if testcase.predecessors %}
+\subsubsection{Predecessors}
+{{ testcase.predecessors }}
+{% endif %}
+
+{% if testcase.required_software or testcase.test_equipment %}
+\subsubsection{Environment Needs}
+{% endif %}
+
+{% if testcase.required_software %}
+\paragraph{Software}
+{{ testcase.required_software }}
+{% endif %}
+
+{% if testcase.test_equipment %}
+\paragraph{Hardware}
+{{ testcase.test_equipment }}
+{% endif %}
+
+{% if testcase.precondition %}
+\subsubsection{Input Specification}
+{{ testcase.precondition }}
+{% endif %}
+
+{% if testcase.postcondition %}
+\subsubsection{Output Specification}
+{{ testcase.postcondition }}
+{% endif %}
+
+\subsubsection{Test Procedure}
+{% if testcase.test_script %}
+    \begin{longtable}[]{p{1.3cm}p{2cm}p{13cm}}
+    %\toprule
+    Step & \multicolumn{2}{@{}l}{Description, Input Data and Expected Result} \\ \toprule
+    \endhead
+    {% for step in testcase.test_script %}
+        {% set rowloop = loop %}
+        {% if step.test_case_key %}
+            {% set step_testcase = testcases_map[step.test_case_key] %}
+            {% for substep in step_testcase.test_script %}
+
+                \multirow{3}{*}{\parbox{1.3cm}{ {{ rowloop.index }}-{{ loop.index }}
+                {\scriptsize from \hyperref[{{ step_testcase.key.lower() }}]
+                {{ curlies(step_testcase.key) }} } } }
+
+                & {\small Description} &
+                \begin{minipage}[t]{13cm}{\scriptsize
+                {{ substep.description }}
+                \vspace{\dp0}
+                } \end{minipage} \\ \cdashline{2-3}
+                & {\small Test Data} &
+                \begin{minipage}[t]{13cm}{\scriptsize
+                {% if substep.test_data %}
+                    {{ substep.test_data }}
+                    \vspace{\dp0}
+                {% endif %}
+                } \end{minipage} \\ \cdashline{2-3}
+                & {\small Expected Result} &
+                {% if substep.expected_result %}
+                    \begin{minipage}[t]{13cm}{\scriptsize
+                    {{ substep.expected_result }}
+                    \vspace{\dp0}
+                    } \end{minipage}
+                {% endif %}
+                \\ \hdashline
+
+            {% endfor %}
+
+        {% else %}
+
+            \multirow{3}{*}{ {{ rowloop.index }} } & Description &
+            \begin{minipage}[t]{13cm}{\footnotesize
+            {{ step.description }}
+            \vspace{\dp0}
+            } \end{minipage} \\ \cline{2-3}
+            & Test Data &
+            \begin{minipage}[t]{13cm}{\footnotesize
+            {% if step.test_data %}
+                {{ step.test_data }}
+                \vspace{\dp0}
+            {% else %}
+                No data.
+                \vspace{\dp0}
+            {% endif %}
+            } \end{minipage} \\ \cline{2-3}
+            {% if step.example_code %}
```

Comment:
yes, because the multirow is setting the high of the first cell, but each line needs to have the necessary & in order to make room for it.
From the other side, the first cell will never be too long, index [+ from LVV-TX if multistep], so the formatting will be fine.